src/libraries/System.Net.Sockets/tests/FunctionalTests/System.Net.Sockets.Tests.csproj

```diff
@@ -55,50 +55,31 @@
     <Compile Include="SocketTaskExtensionsTest.cs" />
     <Compile Include="MulticastOptionTest.cs" />
     <Compile Include="UdpClientTest.cs" />
+    <Compile Include="EnableBroadcastSocketTest.cs" />
     <Compile Include="UnixDomainSocketTest.cs" />
     <!-- Common Sockets files -->
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.cs"
```

Comment:
Can we revert the unrelated formatting changes? I know it's Visual Studio autocorrecting our practices, but still :)

Reply:
Yes, sorry.